JS/auth0.js

```diff
@@ -0,0 +1,33 @@
+const { AUTH0_SECRET, AUTH0_CLIENT_ID, AUTH0_ISSUER_BASE_URL } = process.env;
+const port = process.env.port || 3000;
+let auth_config = undefined;
+
+const secretGiven = () => AUTH0_SECRET != undefined;
+const idGiven = () => AUTH0_CLIENT_ID != undefined;
+const isValidUrl = () => {
+    try {
+      new URL(AUTH0_ISSUER_BASE_URL);
+    } catch (_) {
+      return false;  
+    }
+    if (string.startsWith("https://")) {
+        return true
+    } else {
+        throw new Error("Auth0 issuer base url is not valid, must begin with 'https://'");
```

Comment:
I appreciate you thinking of bad input, but IIRC Auth0's code will validate anything that needs to be https, so I would suggest just removing this specific validation from our code.